src/pages/event-schema.ts

```diff
@@ -5,10 +5,18 @@ const minDate = new Date(2000, 0);
 
 export const eventDateSchema = Joi.date()
   .iso()
+  .allow("")
```

Comment:
empty dates are not valid

Reply:
It won't work if you type a number and then delete:

![image](https://github.com/digital-land/local-plans-timetable/assets/29425751/7e7459dc-64ac-4583-9c17-906c08e46bf0)

I'm not sure what the best aproach is but this is because of the hyphen(s) in the date

Reply:
Thanks for catching this Ronan. I need to look into this a bit further

Reply:
I have added a check to see whether the value only contains hyphens 👍